.github/workflows/ci.yml

```diff
@@ -12,3 +12,28 @@ jobs:
 
       - name: Run tests
         run: python -m unittest test_vectors.py
+
+  format-markdown:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Format Markdown with markdownlint
+        run: |
+          npm install -g markdownlint-cli
+          markdownlint --disable MD013 --fix .
```

Comment:
I did check this out but couldn't really find a command that only checks.
`--fix` doesn't do what we want here, it just fixes things but doesn't return a failed status code if there was something to fix.

Reply:
I've added `git diff` [`--exit-code`](https://git-scm.com/docs/git-diff#Documentation/git-diff.txt---exit-code) which should return 0 when there is nothing to fix, and return 1 when there is.

Reply:
Thank you!